index.html

```diff
@@ -25,6 +25,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+      <nav id="mobile-nav" class="hide">
+        <button onclick="hideMobileMenu();" type="button" class="close">
```

Comment:
- [ ] As the previous reviewer asked, please strictly stick to the original template! The close icon is still not the same as in Figma template: 

## Yours: 

![image](https://user-images.githubusercontent.com/77038610/142750713-5c710544-6b3d-475a-8a71-92ab164698c5.png)


## Original:

![image](https://user-images.githubusercontent.com/77038610/142750704-a38dc3e3-e9d6-4ba9-9cbe-70f507da4175.png)

You can use left side menu in Figma to see the details (eg., colors, font family, font weight, etc.): 

![chrome-capture (3)](https://user-images.githubusercontent.com/77038610/142750794-c2132a61-4994-4fa4-b18b-19eec8973b5d.gif)

You can start from setting color of the icon to:  #67798E !